### ISSUE TITLE
AssertionError running alpha_vantage_api.py: "Got 4850 rows for daily bars table with first day=2001-11-29, last day=2021-03-08, expected 4849 rows"

### ISSUE THREAD

**Issue by rileysparsons** [shlomikushchi/zipline-trader#180]
I often get this error when running 

`export ZIPLINE_TRADER_CONFIG=./config.yaml && export ZIPLINE_ROOT=~/.zipline && python zipline/data/bundles/alpha_vantage_api.py`

```Traceback (most recent call last):
  File "zipline/data/bundles/alpha_vantage_api.py", line 406, in <module>
    show_progress=True,
  File "/Users/rileysteeleparsons/Projects/zipline-trader/zipline/data/bundles/core.py", line 513, in ingest
    pth.data_path([name, timestr], environ=environ),
  File "zipline/data/bundles/alpha_vantage_api.py", line 353, in ingest
    invalid_data_behavior='raise')
  File "/Users/rileysteeleparsons/Projects/zipline-trader/zipline/data/bcolz_daily_bars.py", line 207, in write
    return self._write_internal(it, assets)
  File "/Users/rileysteeleparsons/Projects/zipline-trader/zipline/data/bcolz_daily_bars.py", line 330, in _write_internal
    ).difference(asset_sessions).tolist(),
AssertionError: Got 4850 rows for daily bars table with first day=2001-11-29, last day=2021-03-08, expected 4849 rows.
```

**Comment by shlomikushchi** [comment]
Hi, 
what universe are you using while ingesting?
you are using sqlite right? not postgres?

**Comment by shlomikushchi** [comment]
just ran against SP100:
![image](https://user-images.githubusercontent.com/7924802/110913213-8561c700-831d-11eb-9075-866e433088d3.png)

**Comment by shlomikushchi** [comment]
fixed in v1.6.1
please verify
